kubernetes-docs/mongodb.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Running  MongoDB on vSphere Cloud Provider 
+--- 
+				
+This section describes the steps to create persistent storage for containers to be consumed by MongoDB services on vSAN. After these steps are completed, Cloud Provider will create the virtual disks (volumes in Kubernetes) and mount them to the Kubernetes nodes automatically. The virtual disks are created with the vSAN default policy. 
```

Comment:
YAMLs posted in this md file are not sufficient to deploy mongo db cluster. Please refer to the yamls posted at - https://github.com/vmware/kubernetes/tree/kube-examples/kube-examples/mongodb-shards
and rewrite case study for Mongo DB.